Description: Fix FTBFS on PPC64: inlining failed in call to ‘always_inline’
Author: Dominique Belhachemi <domibel@debian.org>
Forwarded: no
Last-Update: 2025-09-30

--- vtk9.orig/Common/Core/vtkDataArrayMeta.h
+++ vtk9/Common/Core/vtkDataArrayMeta.h
@@ -32,7 +32,7 @@
 #endif
 
 #if (defined(VTK_ALWAYS_OPTIMIZE_ARRAY_ITERATORS) || !defined(VTK_DEBUG_RANGE_ITERATORS)) &&       \
-  !defined(VTK_COMPILER_MSVC)
+  !defined(VTK_COMPILER_MSVC) && !defined(__PPC64__)
 #define VTK_ITER_INLINE VTK_ALWAYS_INLINE
 #define VTK_ITER_ASSUME VTK_ASSUME_NO_ASSERT
 #define VTK_ITER_OPTIMIZE_START VTK_ALWAYS_OPTIMIZE_START
